bridges/snowbridge/pallets/inbound-queue-v2/src/mock.rs

```diff
@@ -161,12 +161,22 @@ pub enum BridgeReward {
 	Snowbridge,
 }
 
+use std::cell::Cell;
+
+thread_local! {
+	pub static REWARD_ALLOCATED: Cell<u128> = Cell::new(0);
+}
```

Comment:
Nit: I think we can use something like:
```
parameter_types! {
	pub static REWARD_ALLOCATED: u128 = 0;
}
```

And then it can be accessed as a storage value using `REWARD_ALLOCATED::get()` and `REWARD_ALLOCATED::set()`.

Also I would rename it to something like `RegisteredRewardsCount`

Reply:
https://github.com/paritytech/polkadot-sdk/pull/8020/commits/1c02ca9d98c7be9e59be8e103f771a3c538ad4c9 and https://github.com/paritytech/polkadot-sdk/pull/8020/commits/308b64189ab320ac0bc61a784b39de3ccec13131 :)